### ISSUE TITLE
README.md options.platforms instructions are fake for CLI

### ISSUE THREAD

**Issue by NottLexa** [nwutils/nw-builder#631]
### Issue Type

- [x] Bug Fix
- [ ] Feature
- [X] Other

nw-builder's README says that options.platforms for CLI can take multiple platforms, they just have to be comma separated:
> **options.platforms**
> Type (CLI): String (comma separated values) ...

Although for Windows it seems like you have to separate platforms by spaces:
`nwbuild * --mode=build --zip=false --buildDir=./dist --platforms=win64 win32`

### Current/Missing Behaviour

If running with `--platforms=win64,win32` / `--platforms win64,win32`:
```
[PROJECT DIRECTORY]\node_modules\nw-builder\lib\index.cjs:107
        throw new Error("Unknown platform " + platform);
        ^

Error: Unknown platform win64,win32
    at [PROJECT DIRECTORY]\node_modules\nw-builder\lib\index.cjs:107:15
    at Array.forEach (<anonymous>)
    at new NwBuilder ([PROJECT DIRECTORY]\node_modules\nw-builder\lib\index.cjs:105:28)
    at nwbuild ([PROJECT DIRECTORY]\node_modules\nw-builder\lib\index.cjs:1088:14)
    at Object.<anonymous> ([PROJECT DIRECTORY]\node_modules\nw-builder\bin\nwbuild.cjs:142:1)
    ...
```

If running with `--platforms=win64 win32` / `--platforms win64 win32`:
```
[ INFO ] Latest Version: v0.67.1
[ INFO ] Using v0.67.1 (sdk)
[ INFO ] Create cache folder in [PROJECT DIRECTORY]\cache\0.67.1-sdk
[ INFO ] Using cache for: win32
[ INFO ] Create cache folder in [PROJECT DIRECTORY]\cache\0.67.1-sdk
[ INFO ] Using cache for: win64
[ INFO ] Create release folder in [PROJECT DIRECTORY]\dist\[name]\win64
[ INFO ] Create release folder in [PROJECT DIRECTORY]\dist\[name]\win32
```

### Expected/Proposed Behaviour

Passing comma separated values, or correct instructions in README.md on [nw-builder github page](https://github.com/nwutils/nw-builder).

### Additional Info

- Operating System: Windows 10 (x64)
- NW.js version: 0.67.1

**Comment by ayushmxn** [comment]
I'm pretty sure the array in the docs is for module usage while for the CLI it is spaces. I'm not sure what the convention is so open to changing the docs or the way CLI arguments are parsed.

**Comment by NottLexa** [comment]
> I'm pretty sure the array in the docs is for module usage while for the CLI it is spaces. I'm not sure what the convention is so open to changing the docs or the way CLI arguments are parsed.

Guess it's better to change parsing CLI arguments, but I am not much of contributor so I can't choose by myself, guess devs have to decide it

**Comment by ayushmxn** [comment]
I'll change it to use commas. Spaces make it more confusing.

**Comment by ayushmxn** [comment]
This is now fixed in `v3.8.3`.
